examples/tf/td3_pendulum.py

```diff
@@ -0,0 +1,83 @@
+"""
+This is an example to train a task with TD3 algorithm.
+
+Here, we create a gym environment InvertedDoublePendulum
+and use a TD3 with 1M steps.
+
+Results:
+    AverageReturn: 250
+    RiseTime: epoch 499
```

Comment:
Have you updated these numbers?

Reply:
I haven't updated it. How do we get these numbers?